core/src/it/scala/org/locationtech/rasterframes/ref/RasterSourceIT.scala

```diff
@@ -71,8 +71,13 @@ class RasterSourceIT extends TestEnvironment with TestData {
 
   describe("GDAL support") {
     it("should read JPEG2000 scene") {
-      val src = RasterSource(localSentinel.toURI)
-      src.readAll().flatMap(_.tile.statisticsDouble).size should be (64)
+      if (sys.env.getOrElse("WITH_GDAL", "true").toBoolean) {
```

Comment:
Could you please change to using this mechanism instead:

https://github.com/s22s/rasterframes/blob/4344e217ca154a08439ccd57cdcf28d7d85fe6bc/core/src/test/scala/astraea/spark/rasterframes/ref/RasterSourceSpec.scala#L113-L129